### PR TITLE
Nepal water level return periods

### PR DESCRIPTION
**Pull request by turnerm**

### CODE REVIEW

File: analyses/npl/07_water_level.md [hunk 1]
```diff
@@ -89,3 +96,42 @@ for station in STATIONS:
...
+    ax.set_title(station)
+    ax.axhline(warning_level, c='C2', label='Warning level')
+    ax.axhline(danger_level, c='C3', label='Danger level')
+    ax.set_xlabel('Return period [years]')
+    ax.set_ylabel('Water level [m]')
+    ax.legend()
+    ax.grid()
+```
+
+```python
```
Comment (hannahker) [inline analyses/npl/07_water_level.md:135]: Could get rid of this!

File: src/utils_general/statistics.py [hunk 2]
```diff
@@ -5,9 +5,20 @@
...
+    rp_var: str,
+    show_plots: bool = False,
+    plot_title: str = "",
 ):
+    """
+    :param df_rp: DataFrame where the index is the year, and the rp_var column contains the maximum value per year
+    :param rp_var: The column with the quantity to be evaluated
+    :param show_plots: Show the histogram with GEV distribution overlaid
+    :param plot_title: The title of the plot
+    :return: Interpolated function that gives the quantity for a give return period
```
Comment (hannahker) [inline src/utils_general/statistics.py:20]: So great to have these docstrings :)

**Review by hannahker — APPROVED** [review]
Merge away!

File: analyses/npl/07_water_level.md (hunk 1 above)
Comment (turnerm) [reply analyses/npl/07_water_level.md:135]: Sorry! :facepalm: I'm the worst, I need to get better at this

File: src/utils_general/statistics.py (hunk 2 above)
Comment (turnerm) [reply src/utils_general/statistics.py:20]: I think we can make them a requirement with some extensions to flake-8... a future issue :)

**Pull request merged**